### PR TITLE
Queue stack

### PR DESCRIPTION
**Pull request by elizaochoa86**
Develop of Queue and Stack Classes

### CODE REVIEW

File: src/main/java/com/learn/java/collections/impl/StackImpl.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Override
+    public void push(int element) {
+        LinkedNode newNode = new LinkedNode(element);
+        if(size == 0){
+            firstNode = newNode;
+        }else{
+            newNode.next = firstNode;
+            firstNode = newNode;
+        }
```
Comment (reyniergt) [inline src/main/java/com/learn/java/collections/impl/StackImpl.java]: I would preffer this: ``firstNode = new LinkedNode(element, firstNode)``

File: src/test/java/com/learn/java/collections/QueueTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+public class QueueTest {
+
+    private Queue queue;
+
+    @Before
+    public void setUp() {
+        //TODO: instanciate the {@link queue}
```
Comment (reyniergt) [inline src/test/java/com/learn/java/collections/QueueTest.java]: remove the `TODO` line

File: src/test/java/com/learn/java/collections/StackTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+public class StackTest {
+
+    private Stack stack;
+
+    @Before
+    public void setUp() {
+        //TODO: instanciate the {@link stack}
```
Comment (reyniergt) [inline src/test/java/com/learn/java/collections/StackTest.java]: remove the `TODO` line

**Review by reyniergt — CHANGES_REQUESTED** [review]

File: src/main/java/com/learn/java/collections/impl/StackImpl.java (hunk 1 above)
Comment (elizaochoa86) [reply src/main/java/com/learn/java/collections/impl/StackImpl.java]: Thanks for your feedback!

**Review by reyniergt — APPROVED** [review]

**Pull request merged**